### DISCUSSION TITLE
v6何时出呀~

### DISCUSSION THREAD

**Discussion by wuranxu** [ant-design/ant-design-pro#10148] [General]
> 看讨论知道项目就一个人在维护了，打算试试v6+umi4，看到现在v6还是beta版本。所以很怕到时候不太兼容，想求个大概日期~

**Comment by jkiss** [comment]
已经用v6写了一个后台系统，比之前的好用多了，第一次用TS，也是比较舒适的。PRO只是对antd+umi的整合，不用太担心兼容问题

**Reply by wuranxu** [reply]
刚才试了一下，navTheme改不了不生效，还是light，antd dark模式也没支持，不知道你那边有没有`遇到`。
我在config.ts和app.ts都改过navTheme，都不行

**Reply by jkiss** [reply]
我直接改的css，默认的样式不满足设计师需求😂
